drivers/sensor/silabs/si7006/si7006.c

```diff
@@ -7,11 +7,11 @@
 
 #include <zephyr/device.h>
 #include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/regulator.h>
 #include <zephyr/drivers/sensor.h>
 #include <zephyr/kernel.h>
 #include <zephyr/sys/byteorder.h>
 #include <zephyr/sys/__assert.h>
-#include <zephyr/drivers/i2c.h>
```

Comment:
Is this not needed for the I2C API calls?

Reply:
It's a duplicate. Look at line 9